deploy/kube-config/riemann/riemann-service.json

```diff
@@ -0,0 +1,36 @@
+{
+  "apiVersion": "v1beta3",
+  "kind": "Service",
+  "metadata": {
+    "labels": {
+      "kubernetes.io/cluster-service": "true",
+      "name": "riemann-heapster"
+    },
+    "name": "riemann-heapster"
+  },
+  "spec": {
```

Comment:
Ok. Shall we ship the UI by default in the same Pod? It will definitely help users. 
Also, can we route data to InfluxDB via Reimann or is it recommended to directly store in InfluxDB (or some other timeseries store)?